app/views/criteria/_boot.js.erb

```diff
@@ -28,4 +28,40 @@
       });
     }
   });
+
+  function add_level(id) {
+    let rubric_form = $('#' + id);
+    let last_level = $('#' + id + " .level").first();
+    let new_level = last_level.clone();
+
+    // use regular expressions to update level numbers
+    let regex = new RegExp(/\d+/);
+    let level_num = $('#' + id + " .level").length;
+    // replace the new level names with the new level number
+    let new_name = $(new_level)[0].childNodes[1];
+    $(new_name).text(I18.t("rubric_criteria.level.name"));
```

Comment:
There's a typo on `I18`. This indicates you didn't test your changes thoroughly before pushing, so please make sure you do this next time.